docs/sample_sheet_schema/schema_specification.md

```diff
@@ -92,7 +93,7 @@ For example:
 ```
 
 will convert the `field` value to a meta value, resulting in the channel `[[id:value, sample:value]...]`
-See [here](https://github.com/nextflow-io/nf-validation/blob/master/plugins/nf-validation/src/testResources/schema_input.json#L8-22) for an example in the sample sheet.
+See [here](https://github.com/nextflow-io/nf-validation/blob/master/plugins/nf-validation/src/testResources/schema_input.json#L10-L25) for an example in the sample sheet.
```

Comment:
Should use permalinks:

```suggestion
See [here](https://github.com/nextflow-io/nf-validation/blob/ce3aef60e5103ea4798375fe6c59bae41b7d2a25/plugins/nf-validation/src/testResources/schema_input.json#L10-L25) for an example in the sample sheet.
```

Reply:
<img width="366" alt="CleanShot 2023-05-31 at 09 43 12@2x" src="https://github.com/nextflow-io/nf-validation/assets/465550/4c815145-4a28-45ce-b711-d4349401af92">